.circleci/config.yml

```diff
@@ -23,6 +23,7 @@ version: 2.1
 
 orbs:
   win: circleci/windows@2.0.0
+  macos: circleci/macos@2.4.0
```

Comment:
Used for rosetta

Reply:
Which xcode version will this contain? We want to make sure we can support a relatively old version of mac

Reply:
It's pinned to `13.4.1` in the executor declaration.